client_example_test.go

```diff
@@ -417,17 +419,19 @@ func ExampleClient_Subscribe() {
 		log.Fatalf("failed to create a stream: %s", err)
 	}
 
-	var stream fauna.Stream
+	var stream fauna.EventSource
 	if err := result.Unmarshal(&stream); err != nil {
 		log.Fatalf("failed to unmarshal the stream value: %s", err)
 	}
 
 	// initiate the stream subscription
-	events, err := client.Subscribe(stream)
+	events, err := client.Stream(stream)
 	if err != nil {
 		log.Fatalf("failed to subscribe to the stream value: %s", err)
 	}
-	defer events.Close()
+	defer func() {
+		_ = events.Close()
+	}()
```

Comment:
Same comment

Reply:
to silence linter warnings about unhandled errors